### ISSUE TITLE
I think there is a code error  happended  in Classifier.py file

### ISSUE THREAD

**Issue by miaomiaoge** [mwdhont/SimCLRv1-keras-tensorflow#2]
In 213 line of Classifier.py file : 

           " for y_t, y_p in zip(data_test, class_test) "

I think it should be : 

           " for y_t, y_p in zip(class_labels, class_test) "

**Comment by mwdhont** [comment]
Hi miamioge,

Thanks for carefully checking the code. I believe however the code is correct as it is. Although the variable names might be a bit confusing, the true labels are stored in the 'data_test' variable, while the 'class_labels' variable is a meant to be a simple list of possible classes.  See the 5th box of 2_model_SimCLR.ipynb, where 'class_labels' is defined as ["cardboard", "glass", "metal", "paper", "plastic"]
